dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/engine/executor/plugin/dynamic/DynamicAsyncTaskExecuteFunction.java

```diff
@@ -142,7 +142,6 @@ private void setOutputParameters() {
 
         List<Property> taskPropertyList = new ArrayList<>(JSONUtils.toList(taskInstance.getVarPool(), Property.class));
```

Comment:
This directly `dynamic` can directly delete.

Reply:
Fixed.